{
    "type": "bundle",
    "id": "bundle--58f5040f-bc7b-4495-b763-f98106ff6863",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--90ebffb8-b4a2-4158-a959-713b12849476",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2024-04-11T00:17:00.683709Z",
            "modified": "2024-04-11T00:17:00.683709Z",
            "name": "CVE-2024-30692",
            "description": "A issue was discovered in ROS2 Galactic Geochelone versions ROS_VERSION 2 and ROS_PYTHON_VERSION 3, allows remote attackers to cause a denial of service (DoS) in the ROS2 nodes.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2024-30692"
                }
            ]
        }
    ]
}